components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -1635,6 +1635,20 @@ const std::string GetAssetRatioBaseURL();
 const base::flat_map<std::string, std::string>& GetAnkrBlockchains();
 // https://docs.rs/solana-program/1.18.10/src/solana_program/clock.rs.html#129-131
 inline constexpr int kSolanaValidBlockHeightThreshold = 150;
+
+inline constexpr char kCSPFrameSrcName[] = "frame-src";
+inline constexpr char kCSPImageSrcName[] = "img-src";
+inline constexpr char kCSPSelf[] = "'self'";
+inline constexpr char kCSPData[] = "data:";
+inline constexpr char kCSPChromeResources[] = "chrome://resources";
```

Comment:
same as above, a constant for `chrome://resources` does not belong in brave_wallet_constants